specification/ai/data-plane/Azure.AI.Projects/preview/2025-05-15-preview/azure-ai-projects.json

```diff
@@ -1801,7 +1801,7 @@
     },
     "Azure.Core.Foundations.InnerError": {
       "type": "object",
-      "description": "An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#handling-errors.",
+      "description": "An object containing more specific information about the error. As per Azure REST API guidelines - https://aka.ms/AzureRestApiGuidelines#handling-errors.",
```

Comment:
minor nit: if users navigate to `https://aka.ms/AzureRestApiGuidelines#handling-errors.` (period at end)  they don't go to the right subsection... consider not ending the URL with period (several spots)